uabot.code-snippets

```diff
@@ -0,0 +1,741 @@
+{
+  "UABot Complete Configuration": {
+    "prefix": "uabotconfig",
+    "scope": "plaintext,json",
+    "body": [
+      "{\r",
+      "\t\"searchEndpoint\": \"https://platform.cloud.coveo.com/rest/search/\",\r",
+      "\t\"analyticsEndpoint\": \"https://usageanalytics.coveo.com/rest/v15/analytics/\",\r",
+      "\t\"orgName\": \"$1\",\r",
+      "\t\"anonymousThreshold\": ${2:Percentage of visits that are anonymous [0..1]},\r",
+      "\t\"dontWaitBetweenVisits\": false,\r",
+      "\t\"timeBetweenVisits\": ${3:Time to wait between the visits in seconds},\r",
+      "\t\"dontWaitBetweenActions\": false,\r",
+      "\t\"timeBetweenActions\": ${4:The time to wait between actions in seconds},\r",
+      "\t\"isWaitConstant\": false,\r",
+      "\t\"partialMatch\": false,\r",
+      "\t\"partialMatchKeywords\": 0,\r",
+      "\t\"partialMatchThreshold\": \"\",\r",
+      "\t\"pipeline\": \"$5\",\r",
+      "\t\"defaultOriginLevel1\": \"$6\",\r",
+      "\t\"defaultOriginLevel2\": \"$7\",\r",
+      "\t\"defaultOriginLevel3\": \"$8\",\r",
+      "\t\"globalfilter\": \"$9\",\r",
+      "\t\"randomGoodQueries\": [\r",
+      "\t],\r",
+      "\t\"randomBadQueries\": [\r",
+      "\t],\r",
+      "\t\"goodQueriesInLanguage\": {\r",
```

Comment:
These are supposed to be a map[string][]string, so something that looks like this:

```
{
"en" : ["query1", "query2"],
"fr" : ["query3", "query4"]
}```

I would remove. Not part of the MVP scenario. Only for specific use cases.